src/html_lustre_converter.gleam

```diff
@@ -184,14 +295,26 @@ fn print_element(
     | "ul"
     | "var"
     | "video" -> {
-      let children = wrap(print_children(children, ws), "[", "]")
+      let children = wrap(print_children(children, ws, False), "[", "]")
       doc.from_string("html." <> tag)
       |> doc.append(wrap([attributes, children], "(", ")"))
     }
 
+    "svg" -> {
+      // TODO: Fix double compute, avoid initial compute if we don't go to SVG land
```

Comment:
What does this mean?

Reply:
Currently, we already compute the attributes at the start of the function, but then when in svg we compute them a second time with the svg-friendly output. I ended up leaving it in when I found no meaningful performance impact, I found it was not necessarily needed. Should have preferenced speaking about it in the PR over leaving a dud TODO :P

Reply:
Can just remove the comment I think